### DISCUSSION TITLE
Unable to login production domain

### DISCUSSION THREAD

**Discussion by Mohanapriya777** [Sunbird-cQube/community#836] [General]
Hi,
We followed the steps in documentation to setup a keycloak setup. But we are unable to login in the production domain(screenshot attached).

<img width="960" alt="Screenshot " src="https://github.com/Sunbird-cQube/community/assets/130358614/39ec95dd-de91-4ed8-ae60-04d679bf8e29">

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha 

Please follow the below steps regarding login issue

1. First go inside querybuilder by using below command
       sudo docker exec -it querybuilder_app sh
2. Then need to install vim by using below command
      apk add vim
3. After that need open .env file by using below command
      vim .env
4. Once open the file add the below command in last line of the .env file
     NODE_TLS_REJECT_UNAUTHORIZED = '0'
5. Once added the above command then restart the querybuilder by using below command
     sudo docker restart querybuilder_app

**Reply by Mohanapriya777** [reply]
Hi ,
The issue is resolved now. Thank you

**Comment by bommalla** [comment]
@Mohanapriya777 @RNarmatha 

Thanks for the update.
